### ISSUE TITLE
SplitView sample needs work for right-alignment

### ISSUE THREAD

**Issue by michael-hawker** [microsoft/WinUI-Gallery#322]
**Is this a totally new sample, an addition to an existing sample page, or a modification of an existing sample page?**
- [ ] New sample
- [ ] Addition to existing sample
- [x] Modification to existing sample

**If this is an addition/modification to an existing sample page, which one?**
SplitView

**Describe the sample**
Change placement to Right:
![image](https://user-images.githubusercontent.com/24302614/72476521-44fecb80-37a2-11ea-8c13-c9558d61f497.png)

It's a bit weird that the content isn't stretched to put the panel on the right-side next to the options.

It get's odder when you change the display mode to `CompactInline` or `CompactOverlay` as the icons don't show up:

![image](https://user-images.githubusercontent.com/24302614/72476623-855e4980-37a2-11ea-8ad8-835db8f4f0a6.png)

(Also changing the DisplayMode to `CompactInline` doesn't appear to work the first time, as `IsPaneOpen` is still on, looks like changing to the overlay modes turns it off as well at the same time, so it's inconsistent here.)

**Comment by chingucoding** [comment]
@stmoy Do you think there is much benefit to having the SplitView be switchable to right placement? Fixing those issues will get quite complicated given how the content is laid out, and I don't think there is much benefit to have a placement toggle here.

If we just show it in placement left, those issues @michael-hawker described here would be gone.

**Comment by michael-hawker** [comment]
@chingucoding I think there's value in the sample gallery showing different layout options to developers/designers so they can see what building blocks they have to build experiences with. Is there a way we can modify the sample content to make this option easier to execute on?

**Comment by chingucoding** [comment]
Well currently we use a ListView to render the items. If we switch from left to right we need to update the ListViewItem style to allow the content render on the right side, and switch to a template which would show the icon on the right, so it is still visible when collapsing.

For the Settings option, I am not sure how we would do that, maybe we need to switch that out too?

The issue here is, essentially we would need to do a complete swap of the content in the Pane when we switch from left to right and vice versa as the positioning would be fundamentally different in those two cases.

**Comment by michael-hawker** [comment]
@chingucoding sounds like another good scenario for my `SwitchPresenter` that I need to add to the Toolkit for everything 😋

I should be gathering all these cases down so I can have different test cases for checking it works for them all...

**Comment by chingucoding** [comment]
Indeed, having a control that would enable switching between two things would make that so much easier to solve! Looking forward to it.

**Comment by bpulliam** [comment]
Do we remove the right-alignment option from the sample? We could do that and keep this issue as an enhancement (to use the SwitchPresenter when that is available).

**Comment by michael-hawker** [comment]
@bpulliam `SwitchPresenter` shipped in 7.0 of the Windows Community Toolkit 🙂 https://learn.microsoft.com/windows/communitytoolkit/controls/switchpresenter

**Comment by AndrewKeepCoding** [comment]
@marcelwgn We can fix this with `VisualStates` and some code-behind.

https://github.com/user-attachments/assets/cf6dc825-9a61-49b9-95bd-44f2bcd4409d

Can I work on this?

**Comment by marcelwgn** [comment]
Sure thing, video looks amazing! Thank you!
